src/Ractive/prototype/fire.js

```diff
@@ -8,6 +8,16 @@ export default function Ractive$fire ( eventName, ...args ) {
 		const ctx = Object.create( proto );
 		Object.assign( ctx, proto );
 		return fireEvent( this, eventName, ctx, args );
+	} else if ( args[0] && typeof args[0] === 'object' && args[0].constructor === Object ) {
```

Comment:
There's an `isObject` in the utils https://github.com/ractivejs/ractive/blob/dev/src/utils/is.js#L25